app/models/school.rb

```diff
@@ -1,3 +1,7 @@
 class School < ActiveRecord::Base
-  has_many :departments, dependent: :destroy
-end
\ No newline at end of file
+  has_many  :departments, dependent: :destroy
+  # every school has a shortname (abbreviation) and full name.
+  # abbreviations are not necessarily unique but full names are
+  validates :shortname, presence: true, uniqueness: false
```

Comment:
I set the uniqueness to false for clarity since on the next line there is another validates call with uniqueness: true. I assumed the shortname of a school was its acronym/nickname, and since two schools can have the same acronym, e.g. BU can be Boston University or Binghamton University, it'd be problematic to have that field be unique.

Reply:
A school is within a university, e.g. "School of Science" or "School of Arts".

Also, it is important in Ruby convention to omit unnecessarily code. `uniqueness: false` does nothing and should not be present.

Reply:
Fair enough, I've deleted the uniqueness parameter. Come to think of it, do we have to have a shortname validation at all?